src/main/config/docker-compose.yaml.ctmpl

```diff
@@ -0,0 +1,22 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{$keyname := printf "secret/dsde/%s/firecloud-ui/docker-compose.yaml" $environment}}
+{{with vault $keyname}}
+
+ui:
+  image: {{.Data.ui_image}}
+  ports:
+    - "80:80"
+    - "443:443"
+    {{.Data.ui_additional_ports}}
+  {{.Data.ui_dns}}
+  {{.Data.ui_volumes}}
+  environment:
+    GOOGLE_CLIENT_ID: {{.Data.env_client_id}}
+    ORCH_URL_ROOT: {{.Data.env_orchestration_url}}
```

Comment:
Okay. There's an example in the README for doing this, which will be wrong when this PR gets merged. Let's close the loop by updating those instructions as part of this PR.